docs/core/gotchas.md

```diff
@@ -0,0 +1,277 @@
+
+A solution after you know it happens is redirect output to a file.
+
+This has happened when running training in a notebook where there are too many lines of output, or if you have a lot (20+) of inline images.
+
++++
+
+# You need at least four atoms for molecules with some models
+
+Gemnet in particular seems to require at least 4 atoms. This has to do with interactions between atoms and their neighbors.
+
+```{code-cell} ipython3
+%%capture
+from ocpmodels.common.relaxation.ase_utils import OCPCalculator
+from ocpmodels.models.model_registry import model_name_to_local_file
+import os
+
+checkpoint_path = model_name_to_local_file('GemNet-OCOC20+OC22', local_cache='/tmp/ocp_checkpoints/')
+
+calc = OCPCalculator(checkpoint_path=checkpoint_path)
+```
+
+```{code-cell} ipython3
+%%capture
+from ase.build import molecule
+import numpy as np
+
+atoms = molecule('H2O')
+atoms.set_tags(np.ones(len(atoms)))
+atoms.set_calculator(calc)
+atoms.get_potential_energy()
+```
+
+# To tag or not?
+
+Some models use tags to determine which atoms to calculate energies for. For example, Gemnet uses a tag=1 to indicate the atom should be calculated. You will get an error with this model 
+
+```{code-cell} ipython3
+%%capture
+from ocpmodels.common.relaxation.ase_utils import OCPCalculator
+from ocpmodels.models.model_registry import model_name_to_local_file
+import os
+
+checkpoint_path = model_name_to_local_file('GemNet-OCOC20+OC22', local_cache='/tmp/ocp_checkpoints/')
+calc = OCPCalculator(checkpoint_path=checkpoint_path)
+```
+
+```{code-cell} ipython3
+%%capture
+atoms = molecule('CH4')
+atoms.set_calculator(calc)
+atoms.get_potential_energy()  # error
+```
+
+```{code-cell} ipython3
+atoms = molecule('CH4')
+atoms.set_tags(np.ones(len(atoms)))  # <- critical line for Gemnet
+atoms.set_calculator(calc)
+atoms.get_potential_energy()
+```
+
+Not all models require tags though. This EquiformerV2 model does not use them. This is another detail that is important to keep in mind.
+
+```{code-cell} ipython3
+from ocpmodels.common.relaxation.ase_utils import OCPCalculator
+from ocpmodels.models.model_registry import model_name_to_local_file
+import os
+
+checkpoint_path = model_name_to_local_file('EquiformerV2 (31M) All+MD', local_cache='/tmp/ocp_checkpoints/')
+
+calc = OCPCalculator(checkpoint_path=checkpoint_path)
+```
+
+```{code-cell} ipython3
+atoms = molecule('CH4')
+
+atoms.set_calculator(calc)
+atoms.get_potential_energy()
+```
+
+# Stochastic simulation results
+
+Some models are not deterministic (SCN/eSCN/EqV2), i.e. you can get slightly different answers each time you run it. An example is shown below. See https://github.com/Open-Catalyst-Project/ocp/issues/563 for more discussion. This happens because a random selection of is made to sample edges, and a different selection is made each time you run it. 
```

Comment:
This link isn't showing up in the docs page, maybe explicitly hyperlink [Issue 563](https://github.com/Open-Catalyst-Project/ocp/issues/563).